integration-tests/reactive-messaging-rabbitmq-vault/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+
+mp.messaging.outgoing.people-out.connector=smallrye-rabbitmq
+mp.messaging.outgoing.people-out.exchange.name=people
+
+mp.messaging.incoming.people-in.connector=smallrye-rabbitmq
```

Comment:
should not be needed anymore.

Reply:
Ok, but I'd rather alter the `RabbitMQConnectorTest` to test if the implicit configuration works. This is specifically to test the dynamic credentials.

Reply:
I've removed the `connector` keys from `RabbitMQConnectorTest` to ensure the implicit configuration is working as expected.